docs/starters.yml

```
@@ -1647,3 +1647,15 @@
     - Comes with React Helmet for adding site meta tags
     - Add Disqus
     - Nice Pagination
+- url: https://master.d2f5ek3dnwfe9v.amplifyapp.com/
+  repo: https://github.com/dabit3/gatsby-auth-starter-aws-amplify
+  description: This Gatsby starter uses AWS Amplify to implement authentication flow for signing up signing in users as well as protected client side routing.
+  tags:
+    - aws
```

This might be a better category, we do the same with netlify
```suggestion
    - AWS Amplify
```